cli/flox/doc/flox-activate.md

```diff
@@ -10,99 +10,103 @@ flox-activate - activate environments
 
 # SYNOPSIS
 
-flox [ `<general-options>` ] activate [ `<options>` ] [ -- `<command>` [ `<argument>` ] ]
+```
+flox [ <general-options> ] activate
+     [-d=<path> | -r=<owner/name>]
+     [ -t ]
+     [ -- <command> [ <arguments> ] ]
+```
 
 # DESCRIPTION
 
-Sets environment variables and aliases, runs hooks and adds environment
-`bin` directories to your `$PATH`. Can be invoked from an interactive
-terminal to launch a sub-shell, non-interactively to produce
-a series of commands to be sourced by your current `$SHELL`,
-or with a command and arguments to be invoked directly.
+Sets environment variables and aliases,
+runs hooks,
+and adds `bin` directories to your `$PATH`.
 
+Launches an interactive subshell when invoked as `flox activate` from an
+interactive shell.
+Launches a subshell non-interactively when invoked with a command and
+arguments.
+May also be invoked as `$(flox activate)` to produce commands to be sourced
+by your current `$SHELL`.
 
+When invoked interactively,
+the shell prompt will be modified to display the active environments,
+as shown below:
+```
+flox [env1 env2 env3] <normal prompt>
+```
 
-# OPTIONS
+When multiple environments are activated each of their shell hooks
+(`hook.script` or `hook.file`)
+are executed in the context of the environment that they come from.
+This means that for each shell hook various environment variables such as
+`PATH`, `MANPATH`, `PKG_CONFIG_PATH`, `PYTHONPATH`, etc,
+are set to the appropriate values for the environment in which the shell
+hook was defined.
 
-```{.include}
-./include/general-options.md
-./include/environment-options.md
-```
+# OPTIONS
 
 ## Activate Options
 
-[ -- `<command>` [ `<argument>` ] ]
+`-- <command> [ <arguments> ]`
 :   Command to run in the environment.
-    Spawns the command in a subshell
-    that does not leak into the calling process.
+    Spawns the command in a subshell that does not leak into the calling
+    process.
+
```

Comment:
missing `-d|--dir` flag

also the order isdifferent than in the USAGE

Reply:
The `-d` and `-r` options are part of `include/environment-options.md`, which Matthew was working on in a separate PR.